### PR TITLE
Add changelog for 9.1.1, 9.0.5, 8.2.8, 8.1.10, 8.0.15

### PR DESCRIPTION
**Pull request by PVince81**
To be merged/deployed after the releases are out (September 20th).

### CODE REVIEW

**Comment by PVince81** [comment]
@RealRancor @agssl @dragotin @danimo FYI

File: page-changelog.php [hunk 1]
```diff
@@ -14,6 +14,75 @@
...
+<li>Sharing: Fix fatal error for users with older existing shares from OC <= 8.2 - <a href="https://github.com/owncloud/core/issue/25933">core/#25933</a></li>
+<li>Sharing: Always allow share owner to increase permissions - <a href="https://github.com/owncloud/core/issue/25542">core/#25542</a></li>
+<li>Sharing: Properly retry federated shares after they were unavailable - <a href="https://github.com/owncloud/core/issue/26037">core/#26037</a></li>
+<li>Sharing: Reallow spaces in federated share autocomplete in share dialog - <a href="https://github.com/owncloud/core/issue/25955">core/#25955</a></li>
+<li>Encryption: OCC command for decryption now doesn't decrypt received shares - <a href="https://github.com/owncloud/core/issue/25599">core/#25599</a></li>
+<li>Files_external: Removed reference magic to avoid potential infinite loops - <a href="https://github.com/owncloud/core/issue/25844">core/#25844</a></li>
+<li>Files_external: Added conditional trace logging for debugging SMB on productive systems - <a href="https://github.com/owncloud/core/issue/25758">core/#25758</a></li>
```
Comment (phil-davis) [inline page-changelog.php]: In the real source of this:
s/productive/production/

File: page-changelog.php [hunk 2]
```diff
@@ -114,6 +183,49 @@
...
+<li>Sharing: Fix public upload issue with quota in some scenarios - <a href="https://github.com/owncloud/core/issue/24751">core/#24751</a></li>
+<li>Sharing: Fix sharing over API when dealing with trailing slashes - <a href="https://github.com/owncloud/core/issue/25464">core/#25464</a></li>
+<li>Sharing: Fix path sent by shared storage hooks, could affect some apps like gallery - <a href="https://github.com/owncloud/core/issue/23620">core/#23620</a></li>
+<li>Sharing: Fix issue where videos did not play from share links with PHP 7 - <a href="https://github.com/owncloud/core/issue/25483">core/#25483</a></li>
+<li>Encryption: OCC command for decryption now doesn't decrypt received shares - <a href="https://github.com/owncloud/core/issue/25599">core/#25599</a></li>
+<li>Files_external: Removed reference magic to avoid potential infinite loops - <a href="https://github.com/owncloud/core/issue/25844">core/#25844</a></li>
+<li>Files_external: Added conditional trace logging for debugging SMB on productive systems - <a href="https://github.com/owncloud/core/issue/25758">core/#25758</a></li>
```
Comment (phil-davis) [inline page-changelog.php]: and here,
s/productive/production

**Comment by PVince81** [comment]
@phil-davis thanks for the review ! Fixed now.

Probably comes from the German "Produktivsystem" :wink:

**Comment by RealRancor** [comment]
Shouldn't the releases be sorted like 9.1.1, 9.0.5, 8.2.8, 8.1.10 and 8.0.15 and afterwards the older releases in the order 9.1.0, 9.0.4, 8.2.7, 8.1.10 and 8.0.15?

Currently its:

9.1.1, 9.1.0, 9.0.5, 9.0.4 etc.

instead of

9.1.1, 9.0.5, ..., 9.1.0, 9.0.4 etc.

File: page-changelog.php [hunk 3]
```diff
@@ -184,6 +323,17 @@
...
+Download: <a href="https://download.owncloud.org/community/owncloud-8.0.15.tar.bz2">owncloud-8.0.15.tar.bz2</a> or <a href="https://download.owncloud.org/community/owncloud-8.0.15.zip">owncloud-8.0.15.zip</a></br>
+MD5: <a href="https://download.owncloud.org/community/owncloud-8.0.15.tar.bz2.md5">owncloud-8.0.15.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-8.0.15.zip.md5">owncloud-8.0.15.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-8.0.15.tar.bz2.sha256">owncloud-8.0.15.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-8.0.15.zip.sha256">owncloud-8.0.15.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-8.0.15.tar.bz2.asc">owncloud-8.0.15.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-8.0.15.zip.asc">owncloud-8.0.15.zip.asc</a></br>
+Packages on our Build Service: <a href="http://software.opensuse.org/download/package?project=isv:ownCloud:community:8.0&package=owncloud">8.0 Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
```
Comment (RealRancor) [inline page-changelog.php]: Packages on our Build Service -> Packages on the OpenSUSE Build Service?

File: page-changelog.php [hunk 4]
```diff
@@ -164,6 +291,18 @@
...
+MD5: <a href="https://download.owncloud.org/community/owncloud-8.1.10.tar.bz2.md5">owncloud-8.1.10.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-8.1.10.zip.md5">owncloud-8.1.10.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-8.1.10.tar.bz2.sha256">owncloud-8.1.10.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-8.1.10.zip.sha256">owncloud-8.1.10.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-8.1.10.tar.bz2.asc">owncloud-8.1.10.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-8.1.10.zip.asc">owncloud-8.1.10.zip.asc</a></br>
+Packages on our Build Service: <a href="http://software.opensuse.org/download/package?project=isv:ownCloud:community:8.1&package=owncloud">8.1 Release Channel</a>, <a href="https://download.owncloud.org/download/repositories/stable/owncloud">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
```
Comment (RealRancor) [inline page-changelog.php]: Packages on our Build Service -> Packages on the OpenSUSE Build Service?

File: page-changelog.php [hunk 5]
```diff
@@ -114,6 +183,49 @@
...
+<li>LDAP: Fix login and logging issue with big avatars by reducing their size - <a href="https://github.com/owncloud/core/issue/25857">core/#25857</a></li>
+<li>LDAP: Prevent triggering email change events at login time for unchanged email - <a href="https://github.com/owncloud/core/issue/25553">core/#25553</a></li>
+<li>LDAP: Hide LDAP admin password in wizard - <a href="https://github.com/owncloud/core/issue/25702">core/#25702</a></li>
+<li>Provisioning API: Fixed issue where subadmins could not change group memberships - <a href="https://github.com/owncloud/core/issue/25496">core/#25496</a></li>
+<li>Activity: Fix owner name processing for received federated shares - <a href="https://github.com/owncloud/core/issue/24938">core/#24938</a></li>
+<li>Updater: Fix web update issue with filesystem apps - <a href="https://github.com/owncloud/updater/issue/371">updater/#371</a></li>
+<li>Password Policy: Properly adjust expiration date when toggling password - password_<a href="https://github.com/owncloud/policy/issue/56">policy/#56</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Enterprise App?

File: page-changelog.php [hunk 6]
```diff
@@ -114,6 +183,49 @@
...
+<li>Firewall: Properly block upload of archive files - <a href="https://github.com/owncloud/firewall/issue/247">firewall/#247</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Enterprise App?

**Comment by PVince81** [comment]
@RealRancor thanks for the review, all your comments make sense. I'll fix it tomorrow.
Somehow my brain sorts versions differently than this page :wink:

**Comment by PVince81** [comment]
@RealRancor fixed, please recheck

**Comment by RealRancor** [comment]
@PVince81 @jnweiger Sorry, havn't noticed this but all links to github are wrong. Each link needs to be:

``https://github.com/owncloud/*/issues/*``

instead of the current:

``https://github.com/owncloud/*/issue/*`` (Note the missing s)

**Comment by PVince81** [comment]
Argh... I should have tested these after using that newly-made magic regexp

**Comment by PVince81** [comment]
will do a new PR

**Comment by PVince81** [comment]
PR here https://github.com/owncloud/owncloud.org/pull/1058

**Pull request merged**